kafka/client.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/Shopify/sarama"
+	"github.com/IBM/sarama"
```

Comment:
Has there been a change in API compatibility? eg: `sarama.RecordHeader`
```golang
// Client kafka interface
type Client interface {
	Produce(ctx context.Context, key, value []byte,
		headers ...sarama.RecordHeader) error
	SendMessage(ctx context.Context, topic string, key, value []byte,
		headers ...sarama.RecordHeader) (partition int32, offset int64, err error)
	AsyncSendMessage(ctx context.Context, topic string, key, value []byte,
		headers ...sarama.RecordHeader) (err error)
	// SendSaramaMessage produce sarama native messages directly
	SendSaramaMessage(ctx context.Context, sMsg sarama.ProducerMessage) (partition int32, offset int64, err error)
```

Reply:
The user needs to replace "github.com/Shopify/sarama" with "github.com/IBM/sarama". 

The "github.com/Shopify/sarama" is no longer in use and we will specify it in the CHANGELOG.